language/Russian/strings.po

```diff
@@ -5750,27 +5811,27 @@ msgstr ""
 
 #: plugins/cheatfind/init.lua:511
 msgid "Automatic"
-msgstr ""
+msgstr "Автоматически"
 
 #: plugins/cheatfind/init.lua:511
 msgid "Manual"
-msgstr ""
+msgstr "Вручную"
 
 #: plugins/cheatfind/init.lua:512
 msgid "Pause Mode"
-msgstr ""
+msgstr "Пауза"
 
 #: plugins/cheatfind/init.lua:519
 msgid "Manually toggle pause when needed"
-msgstr ""
+msgstr "Вручную ставить на паузу по необходимости"
 
 #: plugins/cheatfind/init.lua:522
 msgid "Automatically toggle pause with on-screen menus"
-msgstr ""
+msgstr "Автоматически ставить на паузу при вызове меню"
 
 #: plugins/cheatfind/init.lua:543
 msgid "All slots cleared and current state saved to Slot 1"
-msgstr ""
+msgstr "Все слоты очищены и текущее состояние сохранено в Слоте 1"
```

Comment:
“Slot” in the cheat finder plugin is poor choice of terminology in English, and should not be propagated to translations.  In particular, it causes confusion with slot devices.  In the cheat finder plugin, “slot” refers to a snapshot of memory.  Can you think of a better translation?